week-2-wide-perspective/w02e03-pagination-problem/src/middleware/pagination.middleware.js

```diff
@@ -0,0 +1,17 @@
+const paginationLimitKeys = ["size", "limit", "per_page"];
+const paginationSkipKeys = ["skip", "offset", "ignore"];
```

Comment:
Tutaj bardziej chodziło o to żeby można było ustalić te klucze - samodzielnie w momencie użycia middleware (tam gdzie to wpinasz w route).